app/Http/Controllers/subpagesController.php

```diff
@@ -17,8 +18,23 @@ public function index(string $slug): View|Response
     {
         $parent_page = LandingPageTemplate::getLanding();
         $page = LandingPageTemplate::getSubpage($slug);
+        
         if (empty($page['data'])) {
-            return response()->view('errors.404', [], 404);
+            // Contingency for old visit us pages - not sure if this is the best approach, but it works for now
+            if(str_contains($_SERVER['REQUEST_URI'], 'plan-your-visit')) {
```

Comment:
@JamieC24 this feels like a really fragile approach for this ... can we please catch up on what you're specifically trying to do here and maybe we can figure out a better way to do this?

I don't think merging this in right now is a good idea as I think it's likely to have unintended consequecnes.